doc/source/how-to/releasing.rst

```diff
@@ -246,16 +246,21 @@ auto-generated gRPC interface files from a feature or service that is still
 private, this package should be hosted on a private PyPI repository.
 
 ANSYS, Inc. has a private repository at `PyAnsys PyPI`_. Access is controlled
-via a username and a password: 
-
-+---------------------------------------------+--------------------------------+
-| Credentials for publishing to private PyPI  | Value                          |
-+=============================================+================================+
-| Username                                    | ``__token__``                  |
-+---------------------------------------------+--------------------------------+
-| Password                                    | ``PYANSYS_PYPI_PRIVATE_PAT``   |
-+---------------------------------------------+--------------------------------+
-
+via a username and a password:
+
++---------------------------------------------+-------------------------------------------------------------------------+
+| Credentials for publishing to private PyPI  | Value                                                                   |
++=============================================+=========================================================================+
+| Username                                    | ``__token__``                                                           |
++---------------------------------------------+-------------------------------------------------------------------------+
+| Password                                    | ``PYANSYS_PYPI_PRIVATE_PAT``                                            |
++---------------------------------------------+-------------------------------------------------------------------------+
+| repository-url                              | ``https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/upload``   |
++---------------------------------------------+-------------------------------------------------------------------------+
+
+When running this from the command line using `twine <https://twine.readthedocs.io/>`_,
+be sure to add in `--repository-url`` as an extra option. Otherwise it will
+attempt to send the package to the public PyPI repository.
```

Comment:
```suggestion
be sure to add in `--repository-url`` as an extra option. Otherwise ``twine``
attempts to upload the package to the public PyPI repository.
```